# This message contains arrays describing the positions of the found obstacles.

uint32 numObstacles

float32[] latCenters
float32[] lngCenters
